charts/incubator/gatus/Chart.yaml

```
@@ -0,0 +1,35 @@
+annotations:
+  max_scale_version: 24.04.1
+  min_scale_version: 24.04.0
+  truecharts.org/category: monitoring
+  truecharts.org/max_helm_version: "3.15"
+  truecharts.org/min_helm_version: "3.11"
+  truecharts.org/train: stable
+apiVersion: v2
+appVersion: v5.11.0
```

```suggestion
appVersion: v5.13.1
```

Gets auto corrected later on by a bot. But just best to keep it correct from the start